uPortal-webapp/src/main/webapp/media/skins/respondr/common/common.less

```diff
@@ -30,32 +30,32 @@
  * NOTE:  Do not change the location of Bootstrap sources lightly;  they must
  * be included in the uPortal.war package.
  */
-@import "../../../../webjars/bootstrap/less/bootstrap.less";
+@import url("../../../../webjars/bootstrap/less/bootstrap.less");
```

Comment:
these imports appear to resolve exactly the same, but make it clearer that it is a browser path